nosql/1.0/_index.md

```diff
@@ -46,12 +100,13 @@ template.delete(Car.class, 1L);
 
 
 * [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/governance)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./jakarta-nosql-1.0.0.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./jakarta-nosql-1.0.0.html) (HTML)
 * [Jakarta NoSQL 1.0 Specification Javadoc](./apidocs)
 * Maven coordinates
-  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0-M1](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0-M1/)
-
+  * [jakarta-nosql:jakarta.nosql-api:jar:1.0.0](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0)
+* [Jakarta NoSQL 1.0 TCK](https://www.eclipse.org/downloads/download.php?file=/nosql/1.0/staged/eftl/nosql-1-0-0-tck.zip), [sha](https://download.eclipse.org/nosql/1.0/staged/eftl/nosql-1-0-0-tck.zip.sha256),
+[sig](https://download.eclipse.org/nosql/1.0/staged/eftl/nosql-tck-1.0.0.zip.sig)
```

Comment:
These links should go to the final location, not the staging one

```suggestion
* [Jakarta NoSQL 1.0 TCK](https://download.eclipse.org/jakartaee/nosql/1.0/nosql-1-0-0-tck.zip), [sha](https://download.eclipse.org/jakartaee/nosql/1.0/nosql-1-0-0-tck.zip.sha256),
[sig](https://download.eclipse.org/jakartaee/nosql/1.0/nosql-tck-1.0.0.zip.sig)
```

Reply:
@otaviojava 
Please address @Pandrex247 comments.

Reply:
@kazumura, I saw all the comments; super, thank you. 

My issue was more about when I should do it.

Can I fix it now, or should I wait until the voting process on March 11th?

What I had on my mind was doing it after everybody reviewed it.

Reply:
@otaviojava 
You can fix now.
The files you need to change are:
* _index.md
* jakarta-nosql-1.0.0.html
* jakarta-nosql-1.0.0.pdf

So you don't need to restage TCK artifact.

Reply:
Done!